errors_test.go

```diff
@@ -190,28 +174,24 @@ func TestClassifyError(t *testing.T) {
 // issue13Error returns an error of the type that triggered issue #13,
 // https://github.com/gogama/incite/issues/13.
 func issue13Error(requestID string, statusCode int) error {
-	r := request.Request{
-		RequestID: requestID,
-		HTTPResponse: &http.Response{
+
```

Comment:
nit: Again, unnecessary newline between function prototype and first line of code.